app/code/Magento/InventoryLowQuantityNotificationApi/Test/Api/GetSourceItemConfigurationTest.php

```diff
@@ -14,7 +14,7 @@
 class GetSourceItemConfigurationTest extends WebapiAbstract
 {
     const RESOURCE_PATH = '/V1/inventory/source-item-configuration';
-    const SERVICE_NAME = 'InventoryLowQuantityNotificationApiGetSourceItemConfigurationV1';
+    const SERVICE_NAME = 'inventoryLowQuantityNotificationApiGetSourceItemConfigurationV1';
```

Comment:
resource path should be modified as we don't have SourceItem configuration except LowStock one
```
    <route url="/V1/inventory/source-item-configuration/:sourceCode/:sku" method="GET">
        <service class="Magento\InventoryLowQuantityNotificationApi\Api\GetSourceItemConfigurationInterface" method="execute"/>
        <resources>
            <resource ref="Magento_InventoryApi::source"/>
        </resources>
    </route>
```

all the usages of `source-item-configuration` should be modified to Low Quantity Notification

Reply:
@maghamed, renamed to /V1/inventory/low-quantity-notification.